modules/standard/Sort.chpl

```diff
@@ -471,6 +551,8 @@ The choice of sorting algorithm used is made by the implementation.
   sort. If it is ``true``, it will use an in-place algorithm in order to use
   less memory.
  */
+pragma "last resort"
+@deprecated("The 'sort' function with 'Data' and 'inPlaceAlgorithm' arguments has been deprecated, please use the 'sort' function with an 'x' argument instead")
 proc sort(ref Data: [?Dom] ?eltType, comparator:?rec=defaultComparator,
```

Comment:
If `inPlaceAlgorithm=false`, can this call the new function? that way the logic isn't duplicated

Reply:
I thought about doing that but it's used pretty late in the body of the function, so most of the body would still need to happen when `inPlaceAlgorithm=true`.  The other option would be to write a helper for the parts that will stay the same, but that's some fiddly code movement for something we don't want to live in a helper long term.

Reply:
Sounds good, its not that much code duplication and its temporary anyways